ext/vulnsrc/manual/manual.go

```diff
@@ -12,7 +16,1505 @@ type updater struct {
+					"ExploitabilityScore": 0.0,
+					"ImpactScore":         0.0,
+					"Score":               0.0,
+					"Vectors":             "",
+				},
+				"CVSSv3": map[string]interface{}{
+					"ExploitabilityScore": 3.9,
+					"ImpactScore":         4.0,
+					"Score":               8.6,
+					"Vectors":             "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:C/C:N/I:H/A:N",
+				},
+			},
+		},
+	},
+	// Ubuntu 24.04
+	{
+		Name: "CVE-2024-47076",
+		Namespace: database.Namespace{
+			Name:          "ubuntu:24.04",
+			VersionFormat: dpkg.ParserName,
+		},
+		Description: "cfGetPrinterAttributes5 does not validate IPP attributes returned from an IPP server",
+		Link:        "https://ubuntu.com/security/CVE-2024-47076",
+		// Ubuntu gave this a severity lower than the severity derived from the CVSS score.
+		Severity: database.MediumSeverity,
+		FixedIn: []database.FeatureVersion{
+			{
+				Feature: database.Feature{
+					Name: "libcupsfilters",
+					Namespace: database.Namespace{
+						Name:          "ubuntu:24.04",
+						VersionFormat: dpkg.ParserName,
+					},
+				},
+				Version: "2.0.0-0ubuntu7.1",
+			},
+		},
+		Metadata: map[string]interface{}{
+			"NVD": map[string]interface{}{
+				"PublishedDateTime":    "2024-09-26T16:00Z",
+				"LastModifiedDateTime": "2024-09-27T16:00Z",
+				"CVSSv2": map[string]interface{}{
+					"ExploitabilityScore": 0.0,
+					"ImpactScore":         0.0,
+					"Score":               0.0,
+					"Vectors":             "",
+				},
+				"CVSSv3": map[string]interface{}{
+					"ExploitabilityScore": 3.9,
+					"ImpactScore":         4.0,
+					"Score":               8.6,
+					"Vectors":             "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:C/C:N/I:H/A:N",
+				},
+			},
+		},
+	},
+
+	/********** CVE-2024-47175 **********/
+
+	// Alpine 3.19
+	{
+		Name: "CVE-2024-47175",
+		Namespace: database.Namespace{
+			Name:          "alpine:v3.19",
+			VersionFormat: apk.ParserName,
+		},
+		Description: "CUPS is a standards-based, open-source printing system, and `libppd` can be used for legacy PPD file support. The `libppd` function `ppdCreatePPDFromIPP2` does not sanitize IPP attributes when creating the PPD buffer. When used in combination with other functions such as `cfGetPrinterAttributes5`, can result in user controlled input and ultimately code execution via Foomatic. This vulnerability can be part of an exploit chain leading to remote code execution (RCE), as described in CVE-2024-47176.",
+		Link:        "https://www.cve.org/CVERecord?id=CVE-2024-47175",
+		Severity:    database.HighSeverity,
+		FixedIn: []database.FeatureVersion{
+			{
+				Feature: database.Feature{
+					Name: "cups",
+					Namespace: database.Namespace{
+						Name:          "alpine:v3.19",
+						VersionFormat: apk.ParserName,
+					},
+				},
+				Version: "2.4.9-r1",
+			},
+		},
+		Metadata: map[string]interface{}{
+			"NVD": map[string]interface{}{
+				"PublishedDateTime":    "2024-09-26T16:00Z",
+				"LastModifiedDateTime": "2024-09-27T16:00Z",
```

Comment:
Should this date come from NVD (`2024-09-30`)? Probably doesn't matter. (x3 for other alpines)

Reply:
I got that data on Friday, so it was from 9/27. I can check if it has updated since